### DISCUSSION TITLE
UE4 imports RoadRunner map with missing traffic signs issue

### DISCUSSION THREAD

**Discussion by lemon-lone** [carla-simulator/carla#8040] [General]
At present, I am using RoadRunner to draw a new map and exported the files. fbx and. xodr required by Carla. I moved it to the/Import folder and then used the make import command to build the new map. Everything went smoothly, but I added traffic signs to the map created in RoadRunner. However, when I opened the compiled map with UE, everything else was normal, but there were no traffic signs. Does anyone know about this issue?

**Comment by ShahzebTUMBachelor** [comment]
I have the same problem!

**Reply by lemon-lone** [reply]
Have you solved it? Can you share the technical direction?
